### ISSUE TITLE
Comment function error

### ISSUE THREAD

**Issue by sweord** [VSRonin/QtXlsxWriter#30]
I checkout develop branch for using comment function. But there are always errors. Excel xlsx files was broken after writing comment.

**Comment by VSRonin** [comment]
Yes, that was the straw that broke the camel's back. While implementing that feature I realised the limitations of the design of this library and gave up on developing it further. It's now only in a "static" format.
Basically to make comments work you need informations on the physical position of cells that the library is missing
